### PR TITLE
Unviable portal life forms

### PR DESCRIPTION
**Pull request by NetSysFire**
#### Summary
Content "New portal monsters: unviable life forms"

#### Purpose of change

I actually did not mean to file it already, but this is the easiest way for reviewers.

Not only mi-gos and shoggoths emerge from the portals. Also a lot of things that are simply not at home here and this represents that. Portal fodder, mildly dangerous with some exceptions, as in they can still (try to) bite off your finger if you go poke them.
Also great for flavor.

#### Describe the solution

Mostly a placeholder right now. Tests will fail.

#### Testing

Spawned them. Made them attack me. Looked at natural portal spawns. Butchered them.

### CODE REVIEW

**Comment by Spicyshadow** [comment]
What do these creatures give when butchered? It wouldn´t make sense for them to give tainted or mutant meat.

**Comment by NetSysFire** [comment]
They are your "ordinary" eldritch creatures in that aspect, similar to mi-gos, which do give you tainted meat iirc.
I think the butterfly can just die without leaving a corpse though. I'll find some tainted bug and plant meat for the strange bug and writhing root. The other two are meaty.

**Comment by NetSysFire** [comment]
This *should* be good to go now. I am totally out of energy to continue anyways.

File: data/json/monsters/nether.json [hunk 1]
```diff
@@ -1970,5 +1970,137 @@
...
+    "species": "PLANT",
+    "hp": 20,
+    "speed": 20,
+    "diff": 2,
+    "weight": "1 kg",
+    "volume": "2 L",
+    "default_faction": "nether",
+    "anger_triggers": [ "HURT", "PLAYER_CLOSE" ],
+    "fear_triggers": [ "FIRE" ],
+    "special_attacks": [ { "id": "grab" } ],
```
Comment (Venera3) [inline data/json/monsters/nether.json:2018]: If you want them to be able to grab you you want a non-zero `grab_strength` in the mondef, though if the intention is just to have them flail around that works as well.

File: data/json/monsters/nether.json [hunk 2]
```diff
@@ -1970,5 +1970,137 @@
...
+    "melee_dice": 1,
+    "melee_dice_sides": 2,
+    "melee_damage": [ { "damage_type": "bash", "amount": 1 } ],
+    "anger_triggers": [ "HURT", "PLAYER_CLOSE" ],
+    "morale": 100,
+    "aggression": 20,
+    "bodytype": "blob",
+    "harvest": "mutant_tiny",
+    "zombify_into": "mon_meat_cocoon_tiny",
+    "flags": [ "WARM", "HEARS", "GRABS", "IMMOBILE" ]
```
Comment (Venera3) [inline data/json/monsters/nether.json:2104]: `GRABS` the flag only governs appending "It grabs as an attack" to the x -> e examine window, they'd need the special attack as well.

**Review by Venera3 — COMMENTED** [review]
Really like the flavor on them. Some monstery nitpicks but nothing major.

**Comment by I-am-Erk** [comment]
note that technically mi-go aren't supposed to be portal creatures. They can use portals but they're not extradimensional in origin

**Comment by NetSysFire** [comment]
>note that technically mi-go aren't supposed to be portal creatures. They can use portals but they're not extradimensional in origin

How is that relevant here? I have not touched mi-gos, only a mongroup in which they appear.

Before you start to brainstorm about potential replacements for mi-go, please make that a dedicated issue because it will be lost and forgotten in here.

**Comment by I-am-Erk** [comment]
I'm just mentioning it because you mentioned them, no other reason

File: data/json/monsters/nether.json [hunk 3]
```diff
@@ -1970,5 +1970,140 @@
...
+    "melee_dice_sides": 2,
+    "melee_damage": [ { "damage_type": "bash", "amount": 1 } ],
+    "special_attacks": [ { "id": "grab" } ],
+    "grab_strength": 3,
+    "anger_triggers": [ "HURT", "PLAYER_CLOSE" ],
+    "morale": 100,
+    "aggression": 20,
+    "bodytype": "blob",
+    "harvest": "mutant_tiny",
+    "zombify_into": "mon_meat_cocoon_tiny",
```
Comment (Maleclypse) [inline data/json/monsters/nether.json:2106]:
```suggestion
    "zombify_into": "mon_meat_cocoon_tiny",
```
I'm going ahead and merging this but I could go either way on whether these nether creatures zombify into anything or not.

**Pull request merged**